src/index.md

```diff
@@ -985,35 +985,117 @@ For the purposes of brevity this document will use the term "bug" to refer to bo
 
 _Congratulations!_ Your changeset will now make its way towards a promoted build. When the changeset becomes part of a promoted build, the bug's "Resolved in Build" will have a value of \"b\[1-9\]\[0-9\]&ast;\" to indicate the build number.
 
+# Cloning the JDK
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [openjdk/jdk GitHub project](https://github.com/openjdk/jdk)
+* [Skara Documentation](https://wiki.openjdk.java.net/display/SKARA)
+:::
+
+The complete source code for the JDK is hosted at [GitHub](https://github.com). You can browse the code directly in the [openjdk/jdk GitHub repository](https://github.com/openjdk/jdk), or download the code for offline browsing, editing, and building using `git clone`.
+
+    $ git clone https://github.com/openjdk/jdk.git
+
+If you intend to contribute patches to the JDK, you should first *fork* the JDK repository on GitHub and clone your own fork as shown below. To fork a project on GitHub, go to the [project page](https://github.com/openjdk/jdk) and click the 'Fork' button in the upper right corner, then follow the on screen instructions.
```

Comment:
s/own fork/own personal fork/

This introduces the "personal fork" terminology.

Reply:
Fixed.